src/main/java/pl/javastart/voting/Vote.java

```diff
@@ -11,6 +11,25 @@ public Vote(String voter, Boolean vote) {
         this.vote = vote;
     }
 
+    @Override
+    public String toString() {
+        String voteType;
+
+        try {
+            if (vote) {
+                voteType = "ZA";
+            } else {
+                voteType = "PRZECIW";
+            }
+        } catch (NullPointerException e) {
+            voteType = "WSTRZYMAŁ SIĘ";
+        }
```

Comment:
Nie łap NullPointera, zastąp to ifem - `if (vote == null) ...` - ważne, żeby ten if był jako pierwszy